doc/reference/guidelines.rst

```diff
@@ -0,0 +1,48 @@
+.. highlight:: python
+
+=================
+Coding Guidelines
+=================
+
+Python
+======
+
+- PEP8 ignore E501
+- prefer % over format
+- try to avoid generators and decorators
+
+
+File naming
+===========
+
+Split the business logic by sets of models, in each sets select a main
+model, this model gives its name to the set. If there is only one set of module
+it's name is the same as the module name.
+
+For each set named <main_model> the following files may be created:
+
+data/<main_model>.xml
+models/<main_model>.py
+views/<main_model>_templates.xml
+views/<main_model>_view.xml
```

Comment:
If you use plural for templates, then you should use plural for views as well:

    views/<main_model>_templates.xml
    views/<main_model>_views.xml